### ISSUE TITLE
New item screen, item icon disappears

### ISSUE THREAD

**Issue by catskull** [andwn/cave-story-md#19]
I think this happens if the screen is still moving when an item is picked up. The item icon will just disappear. If I wait until the screen is not moving, the icon stays. Here's a picture:
![image](https://cloud.githubusercontent.com/assets/6233563/16898245/cf199508-4b8c-11e6-9992-7f1070434bc1.png)

**Comment by andwn** [comment]
This is related to sprite sorting. I am waiting on upstream functionality because I want to work on other parts of the game.

**Comment by catskull** [comment]
Duplicate of #6, closing for now.
